CONTRIBUTING.md

```diff
@@ -69,6 +69,36 @@ tox run -e integration   # integration tests
 tox                      # runs 'lint' and 'unit' environments
 ```
 
+## Documentation
+
+Product documentation is stored in [Discourse](https://discourse.charmhub.io/t/charmed-kafka-k8s-documentation/10296) and published on Charmhub and the Canonical website via Discourse API. 
+The documentation in this repository under the `docs` folder is a mirror synched by [Discourse Gatekeeper](https://github.com/canonical/discourse-gatekeeper), that takes care of automatically raising and updating a PR whenever changes to the content on Discourse are made.
+Although Discourse content can be edited directly, unless the modifications are trivial and obvious (typos, spellings, formatting) we generally recommend to follow a review process:
+
+1. Create a branch (either in the main repo or in a fork) from the current `main` and modify documentation files as necessary.
+2. Raise a PR against the `main` to start the review process, and conduct the code review within the PR.
+3. Once the PR is approved and all comments are addressed, the PR should NOT be merged directly! All the modifications should be applied to Discourse manually. If needed, new Discourse topics can be created, and referenced in the navigation table of the main index file on Discourse.
+4. Discourse Gatekeeper will raise a new PR or add new commits to an open Discourse PR, tracking the `discourse-gatekeeper/migrate` branch. The [sync_docs.yaml](https://github.com/canonical/kafka-k8s-operator/actions/workflows/sync_docs.yaml) GitHub Actions provides further details on the Gatekeeper integration that can be run (a) in a scheduled fashion every night; (b) as a part of pull request CI, and (c) can be triggered manually. If new topics are referenced in the main index file on Discourse, these will be added to `docs/index.md` and the new topics pulled from Discourse.
+5. Once Gatekeeper has raised a new or updated an existing PR, feel free to close the initial PR manually created in step 2, with a comment referring to the PR created by Gatekeeper. If the initial PR was referring to a ticket, add the ticket to either the title or the description of the GateKeeper PR.
+
+### Terminology
+
+Apache®, [Apache Kafka, Kafka®](https://kafka.apache.org/), [Apache ZooKeeper, ZooKeeper™](https://zookeeper.apache.org/) and their respective logos are either registered trademarks or trademarks of the [Apache Software Foundation](https://www.apache.org/) in the United States and/or other countries.
+
+For documentation in this repository the following conventions are applied (see the table below).
+
+| Full form | Alternatives | Incorrect examples |
+| -------- | ------- | ------- |
+| Apache Kafka | | Kafka |
+| Charmed Apache Kafka K8s | | Charmed Kafka K8s, Charmed Kafka |
+| Kafka Connect | | Kafka connect |
+| Kafka Brokers | | Kafka brokers|
```

Comment:
**todo:** Remove L95. `Kafka brokers` is correct.

Reply:
Both options are viable as long as we are consistent in using only one of them. That's not for me to decide what option is better. Can you please discuss that with @grobbie and @deusebio ?

Reply:
I really don't have strong opinions on this, as long as we are consistent, as Vladimir is also suggesting. I would go in the following order:
1. @grobbie 
2. @izmalk 
If neither of you have any opinion, I'm happy with what @marcoppenheimer suggests as well.

Reply:
I've discussed this with @grobbie and we've settled with `Apache Kafka brokers`.

Reply:
```suggestion
| Apache Kafka brokers | | Kafka brokers, Apache Kafka Brokers |
```